run.py

```diff
@@ -40,10 +51,35 @@ def create_app(config_filename: str) -> Flask:
 
 @application.before_first_request
 def create_tables():
+
     from app.database.sqlalchemy_extension import db
 
+    from app.database.models.user import UserModel
+    from app.database.models.mentorship_relation import (
+        MentorshipRelationModel,
+    )
+    from app.database.models.tasks_list import TasksListModel
+    from app.database.models.task_comment import TaskCommentModel
+
     db.create_all()
 
 
+@application.shell_context_processor
+def make_shell_context():
+    return {
+        "db": db,
+        "UserModel": UserModel,
+        "MentorshipRelationModel": MentorshipRelationModel,
+        "TaskListModel": TasksListModel,
+        "TaskCommentModel": TaskCommentModel,
+        "OrganizationModel": OrganizationModel,
+        "ProgramModel": ProgramModel,
+        "UserExtensionModel": UserExtensionModel,
+        "PersonalBackgroundModel": PersonalBackgroundModel,
+        "MentorshipRelationExtensionModel": MentorshipRelationExtensionModel,
+    }
+
+
 if __name__ == "__main__":
+    # if we use docker then we can also run it on the default 5000 port
```

Comment:
can we keep both. So, give options for people who want to run locally as well using port 4000. You can comment that option out.